frontend/views/layout/Layout.style.ts

```diff
@@ -6,6 +6,8 @@ export const Main = styled.main`
     /* TODO: figure out breakpoints, these are an idea for now */
 
     /*  4 padding / 0 / 4 padding + 6 navbar */
+
+    /* I hid it because I don't know what it was */
```

Comment:
you can delete this if this comment is outdated (so we don't get confused later)